RepositoryCompilerTests/Unit/CaDETCodeModelTests.cs

```diff
@@ -123,7 +124,7 @@ public void Calculates_tight_class_cohesion()
             var class7 = classes.Find(c => c.Name.Equals("Class7"));
             var class8 = classes.Find(c => c.Name.Equals("Class8"));
 
-            class6.Metrics.TCC.ShouldBe(0.6666666666666666);
+            Math.Round((double)class6.Metrics.TCC, 2).ShouldBe(0.67);
```

Comment:
This does not solve the issue - The goal is to round the result of the actual TCC calculation so that our model contains a number with up to two digits. The test should remain as it were, where we want to see if "class6.Metrics.TCC.ShouldBe(0.67)"

Reply:
Resolved, now model contains a number with two digits.